src/util.js

```diff
@@ -197,3 +191,58 @@ export const importantify = (string) =>
     string.replace(
         IMPORTANT_RE,
         (_, base) => base + " !important;");
+
+const getBrowserProperties = () => {
+    if (!getBrowserProperties.availableStyles) {
+        getBrowserProperties.availableStyles = {};
+        const styles = Object.keys(window.getComputedStyle(document.documentElement, ''));
+        for (let i = 0; i < styles.length; i++) {
+            const style = styles[i];
+            if (isNaN(Number(style))) {
+                getBrowserProperties.availableStyles[style] = style;
+            }
+        }
+    }
+    return getBrowserProperties.availableStyles;
+};
+
+const stylePrefixes = ['Moz', 'webkit', 'ms', 'O'];
+const getVendorPrefix = (property) => {
+    const validProperties = getBrowserProperties();
+
+    const validatedProp = validProperties[property];
+    if (validatedProp) {
+        return validatedProp;
+    }
+    const capitalProp = property[0].toUpperCase() + property.substr(1);
+    for (let i = 0; i < stylePrefixes.length; i++) {
+        const prefix = stylePrefixes[i];
+        const prefixedProperty = `${prefix}${capitalProp}`;
+        if (validProperties[prefixedProperty]) {
+      // learn which styles the browser likes
+            validProperties[validatedProp] = prefixedProperty;
```

Comment:
I understand that. But following the control flow of the code, at this line (223), the value of `validatedProp` is falsy (an empty string (`""`), zero (0), `false`, `undefined` or `null`). Doing simple substitution, we arrive at either one of the following lines of code:

```
validProperties[undefined] = prefixedProperty;
validProperties[""] = prefixedProperty;
validProperties[0] = prefixedProperty;
validProperties[false] = prefixedProperty;
```

Is that really what you intended to do?